Gemfile.lock

```diff
@@ -403,10 +403,11 @@ GEM
     zeitwerk (2.5.4)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```

Comment:
@skateman could this change be a problem for any reason?

Reply:
Better safe than sorry, @marleystipich2 can you please revert this line to its original version?

Reply:
Yeah, good catch!